LunaMary/src/MultipleChoiceQuestion.java

```diff
@@ -0,0 +1,91 @@
+import java.util.Map;
+
+
+class MultipleChoiceQuestion implements IQuestion {
+
+    private String title;
+    private Map<String, Option> optionMap;
+
+    public MultipleChoiceQuestion(String title, Map<String, Option> optionMap) {
+        this.title = title;
+        this.optionMap = optionMap;
+    }
+
+    public MultipleChoiceQuestion(String title) {
+        this.title = title;
+        this.optionMap = null;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public void setTitle(String title) {
+        this.title = title;
+    }
+
+    public Map<String, Option> getOptionMap() {
+        return optionMap;
+    }
+
+
+    public void answer(String resp) {
+
+    }
+
+    public String display() {
+
+        String result = "P. " + getTitle() + "\n";
+        result += displayOptions(optionMap);
+
+        return result;
+    }
+
+    public String displayOptions(Map<String, Option> optionMap) {
+        String result = "";
+        for (Map.Entry<String, Option> entry : optionMap.entrySet()) {
+            String sigla = entry.getKey();
+            Option option = entry.getValue();
+            result += " " + sigla + ": " + option.getOption() + "\n";
+        }
+
+        return result;
+    }
+
+    public Integer evaluate() {
+        return 1;
+    }
+
+    @Override
+    public void saveAnswer(String resp) {
+        int i = 0;
+//        while (i < optionList.size()) {
+//            if (resp.equalsIgnoreCase(optionList.get(i).getSigla().toString())) {
+//                optionList.get(i).setStatus(true);
+//                i = optionList.size();
+//            } else
+//                i++;
+//        }
+
+
+    }
+
```

Comment:
You are missing addOption method that receives an option. And adds it to your option map